omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/ProgramAttributeTypeResource2_2.java

```diff
@@ -39,31 +46,47 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 		}
 		return super.getRepresentationDescription(rep);
 	}
-	
+
 	@Override
 	public ProgramAttributeType getByUniqueId(String uuid) {
 		return Context.getProgramWorkflowService().getProgramAttributeTypeByUuid(uuid);
 	}
-	
+
 	@Override
 	public ProgramAttributeType newDelegate() {
 		return new ProgramAttributeType();
 	}
-	
+
 	@Override
 	public ProgramAttributeType save(ProgramAttributeType programAttributeType) {
 		return Context.getProgramWorkflowService().saveProgramAttributeType(programAttributeType);
 	}
-	
+
 	@Override
 	public void purge(ProgramAttributeType programAttributeType, RequestContext requestContext) throws ResponseException {
 		Context.getProgramWorkflowService().purgeProgramAttributeType(programAttributeType);
 	}
-	
+
 	@Override
 	protected NeedsPaging<ProgramAttributeType> doGetAll(RequestContext context) throws ResponseException {
 		return new NeedsPaging<ProgramAttributeType>(Context.getProgramWorkflowService()
 		        .getAllProgramAttributeTypes(),
 		        context);
 	}
+
+	@PropertyGetter("concept")
+	public Object getConcept(ProgramAttributeType delegate) {
+		if (!OpenmrsUtil.nullSafeEquals(delegate.getDatatypeClassname(), ConceptDatatype.class.getCanonicalName()))
+			return null;
+		else {
```

Comment:
The else is not needed. The return statement on top exists this method if the condition is true.
You can now write the remaining logic without the else { } .

Reply:
@gsluthra addressed the comment